src/lib.rs

```diff
@@ -1,5 +1,6 @@
 mod arithmetic;
 pub mod hash_to_curve;
+// TODO: remove this and use traits defined in axiom_pairing instead.
```

Comment:
Would this be done in a separate PR? If yes, can we file an issue for this task to keep a track?

Reply:
I don't think we have issues enabled on this repo and I cannot see a way for me to enable them. I assume I don't have the right permissions.